components/test_raftstore/src/util.rs

```diff
@@ -396,14 +400,14 @@ pub fn check_raft_cmd_request(cmd: &RaftCmdRequest) -> bool {
     is_read
 }
 
-pub fn make_cb(
+pub fn make_cb<EK: KvEngine>(
```

Comment:
```suggestion
// So that not a lot of code change is needed
pub fn make_cb(...) {
    make_cb2::<RockEngine>()
}
pub fn make_cb2<EK: KvEngine>(
```

Reply:
Done